### PR TITLE
feat(apollo-hooks): simplify params and use @apollo/react-hooks

### LINKED ISSUE #3: Use official hooks package

### PR DESCRIPTION
**Pull request by fakenickels**
Closes #3

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -12,17 +12,17 @@
   ],
   "author": "",
   "license": "MIT",
-  "dependencies": {
```
Comment (fakenickels) [inline package.json]: this saves lifes

File: src/Mutation.re [hunk 2]
```diff
@@ -33,12 +33,12 @@ module Make = (Config: Config) => {
     }) =
     "useMutation";
 
-  let use = (~options=?, ()) => {
+  let use = (~variables=?, ~client=?, ()) => {
```
Comment (fakenickels) [inline src/Mutation.re]: a bit more generated JS code by BS but a better DX

Comment (georgelima) [reply src/Mutation.re]: nice!

**Comment by fakenickels** [comment]
wrong base

**Pull request closed without merging**